### DISCUSSION TITLE
Recreate the ocrized document

### DISCUSSION THREAD

**Discussion by ep0p** [mindee/doctr#1633] [Q&A]
I have been looking into solutions to be able to re-create the original document [using ocrmypdf](https://github.com/mindee/doctr/discussions/1497) (pdf/a is not a good solution for me)

i was wondering if you have some tips for well formatting the text coming from the hocr. At the moment i have no spaces between words 
![Screenshot from 2024-06-05 10-34-35](https://github.com/mindee/doctr/assets/168719294/433d4f62-ce4d-4a3e-8971-c2c241b0c3eb)

Or if there are some other solutions i would appreciate them.

**Comment by felixdittrich92** [comment]
Hi @ep0p :wave:,

Have you tried our reconstruction ?

```
import matplotlib.pyplot as plt

synthetic_pages = result.synthesize()
plt.imshow(synthetic_pages[0]); plt.axis('off'); plt.show()
```

There is an open ticket to improve the quality but maybe it fits your needs already :)
ref.: #1528

**Comment by ep0p** [comment]
Hi @felixdittrich92,

I have indeed tested your solution, but unfortunately it does not solve my problem. My goal is to be able to exactly recreate the original document .
 
Thank you for your answer.
